packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -266,13 +269,26 @@ export default async function pluginContentBlog(
             JSON.stringify(metadata, null, 2),
           );
 
+          const blogMetadataPath = await createData(
+            `${docuHash(`${metadata.source}-blogMetadata`)}.json`,
+            JSON.stringify(
+              {
+                baseBlogPermalink,
+                blogTitle,
+              },
+              null,
+              2,
+            ),
+          );
+
```

Comment:
we'd want to create one bundle/path and reuse it for all blog posts here

since it's always the same data it's not worth it to create one new bundle per blog post: we can optimize and make it fetch the bundle only once for all the blog posts

Reply:
Good catch - fixed!